tests/PHPUnit/PredisConnectionTestCase.php

```diff
@@ -539,9 +541,9 @@ protected function assertNonPersistentConnection($resource): void
      *
      * @return NodeConnectionInterface
      */
-    protected function createConnection(bool $initialize = false): NodeConnectionInterface
+    protected function createConnection(bool $initialize = false, bool $noAuth = false): NodeConnectionInterface
```

Comment:
As mentioned above, I think we should have both cases, with and without pass

Reply:
Resolved